react-app/src/api/stakingAPI.ts

```diff
@@ -293,6 +293,15 @@ export const useStakingAPI = (): IStakingAPI => {
       );
 
       const commissionRaw = validator.commission;
+
+      let commissionUpdateTime: Date | null = null;
+      if (commissionRaw?.updateTime) {
+        commissionUpdateTime = new Date(0);
+        commissionUpdateTime.setUTCSeconds(
+          commissionRaw.updateTime.seconds.toNumber()
+        );
+      }
```

Comment:
`comissionUpdateTime` is in the type of Timestamp (from cosmjs-type), in which Timestamp `Represents seconds of UTC time since Unix epoch`, so this few lines is to convert that back to a js date object.